plugins/cue_points/quiz/QuizPlugin.php

```diff
@@ -502,7 +502,19 @@ protected function getQuestionCountByQusetionIds($objectIds)
 	protected function getAnswerCountByUserIds($objectIds)
 	{
 		$c = $this->createGetCuePointByUserIdsCriteria($objectIds);
-		$numOfAnswers = CuePointPeer::doCount($c);
+		$numOfAnswers = 0;
+		$answers = CuePointPeer::doSelect($c);
+		foreach ($answers as $answer)
+		{
+			/**
+			 * @var AnswerCuePoint $answer
+			 */
+			$quizUserEntryId = $answer->getQuizUserEntryId();
+			if ($this->isQuizUserEntrySubmitted($quizUserEntryId))
```

Comment:
Since isQuizUserEntrySubmitted method calls UserEntryPeer::retrieveByPK, you better implement it here and load all UserEntry objects in one DB request.